namespace py services.types
namespace java services.types
namespace go services.types

typedef string UserID

struct UserInfo {
    10: string first_name
    20: string last_name
}
